pkg/internalapis/cnsoperator/triggercsifullsync/v1alpha1/triggercsifullsync_types.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2021 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TriggerCsiFullSyncSpec defines the desired state of TriggerCsiFullSync
+// +k8s:openapi-gen=true
+type TriggerCsiFullSyncSpec struct {
+	// TriggerSync gives an option to trigger full sync on demand.
+	// Inorder to trigger a full sync, user can set any random value.
+	TriggerSync int `json:"triggerSync"`
+}
+
+// TriggerCsiFullSyncStatus contains the status for a TriggerCsiFullSync
+// +k8s:openapi-gen=true
+type TriggerCsiFullSyncStatus struct {
+	// InProgress indicates whether a CSI full sync is in progress.
+	// If full sync is completed this field will be unset.
```

Comment:
Will it be unset or it will be set to false?

Reply:
unset. By default unset in golang has a value false.

Reply:
I mean in the CR - would we see it as false too?

Reply:
yes false.